src/System.Security.Cryptography.Algorithms/src/ApiCompatBaseline.net463.txt

```diff
@@ -3,6 +3,8 @@ MembersMustExist : Member 'System.Security.Cryptography.ECDsa.Create(System.Secu
 MembersMustExist : Member 'System.Security.Cryptography.ECDsa.Create(System.Security.Cryptography.ECParameters)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Security.Cryptography.ECDsa.ExportExplicitParameters(System.Boolean)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Security.Cryptography.ECDsa.ExportParameters(System.Boolean)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.Security.Cryptography.ECDsa.FromXmlString(System.String, System.Security.Cryptography.ECKeyXmlFormat)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.Security.Cryptography.ECDsa.ToXmlString(System.Security.Cryptography.ECKeyXmlFormat)' does not exist in the implementation but it does exist in the contract.
```

Comment:
Ooh, no.  These aren't supported.

ToXmlString(ECKeyXmlFormat) is only defined on ECDsaCng and ECDiffieHellmanCng.  Non-virtual in either case, and I didn't seem to move them.

So they shouldn't be showing up in a netstandard2.0 diff, and shouldn't be added on the base class.